frontend/packages/kubevirt-plugin/src/components/modals/create-vm-wizard.tsx

```diff
@@ -80,7 +81,21 @@ export const openCreateVmWizard = (activeNamespace, createTemplate = false) => {
     };
   };
 
-  const launcher = createModalResourceLauncher(CreateVmWizard, resources, resourcesToProps);
+  type Props = {
+    k8sCreate: (kind: any, data: any, opts?: {}) => Promise<any>;
+    k8sGet: (kind: any, name: any, ns: any, opts: any) => Promise<any>;
+    k8sPatch: (kind: any, resource: any, data: any) => Promise<any>;
+    k8sKill: (kind: any, resource: any, opts?: {}, json?: any) => Promise<any>;
```

Comment:
- kind and resource can have the respective types: `K8sKind` & `K8sResourceKind`
- opts should be also any, because it will not register any property like this.
- Promise can return also K8sResourceKind (not sure about the `k8sKill`)
- you can make the K8sResourceKind generic if you want to be the most correct

Reply:
@suomiy Thanks, that's part of #1822, 

I'm not sure things will stay the same in #1822 ... I just added these lines here so it will run and I can make some screen shot :-)

@rawagner does it make sense to edit the vm-wizard files in this PR ?

Reply:
@suomiy - should have separated the commits to `vm-fixes` and `vm-template things` ... , sorry :white_flower:

Reply:
separated the commits to things related to 1822, and thing related to this PR :peace_symbol: :beetle:

Reply:
+1